src/java/org/apache/cassandra/metrics/InternodeInboundMetrics.java

```diff
@@ -20,27 +20,29 @@
 import com.codahale.metrics.Gauge;
 import org.apache.cassandra.locator.InetAddressAndPort;
 import org.apache.cassandra.net.InboundMessageHandlers;
-import org.apache.cassandra.metrics.CassandraMetricsRegistry.MetricName;
+import static org.apache.cassandra.metrics.CassandraMetricsRegistry.Metrics;
 
 /**
  * Metrics for internode connections.
  */
 public class InternodeInboundMetrics
 {
-    private final MetricName corruptFramesRecovered;
-    private final MetricName corruptFramesUnrecovered;
-    private final MetricName errorBytes;
-    private final MetricName errorCount;
-    private final MetricName expiredBytes;
-    private final MetricName expiredCount;
-    private final MetricName pendingBytes;
-    private final MetricName pendingCount;
-    private final MetricName processedBytes;
-    private final MetricName processedCount;
-    private final MetricName receivedBytes;
-    private final MetricName receivedCount;
-    private final MetricName throttledCount;
-    private final MetricName throttledNanos;
+    public final Gauge<Long> corruptFramesRecovered;
```

Comment:
Most of the changes here are not necessary and seem to increase the severity of the conflicts when we merge/rebase on the OSS in the future.

Anyway, protected metric names would have been enough, together with sth along
```
gauge = CassandraMetricsRegistry.Metrics.getMetrics().get(metricName)
```
in CNDB